### ISSUE TITLE
Unable to build using base images python:3.11.0-slim-bullseye

### ISSUE THREAD

**Issue by jonasbn** [rojopolis/spellcheck-github-actions#131]
The `lxml` requirement will not build with this newer base image.

```
python-3.11.0-slim-bullseye ✗ $ docker build -t jonasbn/github-action-spellcheck:latest .
[+] Building 8.3s (11/12)
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 964B                                       0.0s
 => [internal] load .dockerignore                                          0.0s
 => => transferring context: 34B                                           0.0s
 => [internal] load metadata for docker.io/library/python:3.11.0-slim-bul  1.6s
 => [auth] library/python:pull token for registry-1.docker.io              0.0s
 => [internal] load build context                                          0.0s
 => => transferring context: 106B                                          0.0s
 => [1/7] FROM docker.io/library/python:3.11.0-slim-bullseye@sha256:f8cc8  0.0s
 => CACHED [2/7] RUN apt-get update && apt-get install -y     aspell aspe  0.0s
 => CACHED [3/7] COPY entrypoint.sh /entrypoint.sh                         0.0s
 => CACHED [4/7] COPY requirements.txt /requirements.txt                   0.0s
 => CACHED [5/7] COPY spellcheck.yaml /spellcheck.yaml                     0.0s
 => ERROR [6/7] RUN pip3 install -r /requirements.txt                      6.7s
------
 > [6/7] RUN pip3 install -r /requirements.txt:
#11 1.237 Collecting backrefs==5.0.1
#11 1.338   Downloading backrefs-5.0.1-py39-none-any.whl (376 kB)
#11 1.405      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 376.3/376.3 kB 5.8 MB/s eta 0:00:00
#11 1.433 Collecting beautifulsoup4==4.9.3
#11 1.449   Downloading beautifulsoup4-4.9.3-py3-none-any.whl (115 kB)
#11 1.463      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 115.8/115.8 kB 9.4 MB/s eta 0:00:00
#11 1.592 Collecting bracex==2.1.1
#11 1.611   Downloading bracex-2.1.1-py3-none-any.whl (10 kB)
#11 1.636 Collecting html5lib==1.1
#11 1.653   Downloading html5lib-1.1-py2.py3-none-any.whl (112 kB)
#11 1.663      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 112.2/112.2 kB 13.5 MB/s eta 0:00:00
#11 1.795 Collecting lxml==4.9.1
#11 1.817   Downloading lxml-4.9.1.tar.gz (3.4 MB)
#11 2.056      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.4/3.4 MB 14.3 MB/s eta 0:00:00
#11 2.249   Preparing metadata (setup.py): started
#11 2.582   Preparing metadata (setup.py): finished with status 'done'
#11 2.619 Collecting Markdown==3.3.4
#11 2.632   Downloading Markdown-3.3.4-py3-none-any.whl (97 kB)
#11 2.646      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 97.6/97.6 kB 8.2 MB/s eta 0:00:00
#11 2.770 Collecting pymdown-extensions==8.2
#11 2.796   Downloading pymdown_extensions-8.2-py3-none-any.whl (207 kB)
#11 2.820      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 207.7/207.7 kB 9.1 MB/s eta 0:00:00
#11 2.930 Collecting pyspelling==2.8.1
#11 2.951   Downloading pyspelling-2.8.1-py3-none-any.whl (45 kB)
#11 2.958      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 45.3/45.3 kB 6.5 MB/s eta 0:00:00
#11 3.000 Collecting PyYAML==5.4.1
#11 3.017   Downloading PyYAML-5.4.1.tar.gz (175 kB)
#11 3.039      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 175.1/175.1 kB 9.1 MB/s eta 0:00:00
#11 3.109   Installing build dependencies: started
#11 4.909   Installing build dependencies: finished with status 'done'
#11 4.910   Getting requirements to build wheel: started
#11 5.329   Getting requirements to build wheel: finished with status 'done'
#11 5.331   Preparing metadata (pyproject.toml): started
#11 5.478   Preparing metadata (pyproject.toml): finished with status 'done'
#11 5.504 Collecting six==1.16.0
#11 5.522   Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)
#11 5.553 Collecting soupsieve==2.2.1
#11 5.572   Downloading soupsieve-2.2.1-py3-none-any.whl (33 kB)
#11 5.689 Collecting wcmatch==8.2
#11 5.703   Downloading wcmatch-8.2-py3-none-any.whl (41 kB)
#11 5.710      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 41.2/41.2 kB 7.8 MB/s eta 0:00:00
#11 5.731 Collecting webencodings==0.5.1
#11 5.747   Downloading webencodings-0.5.1-py2.py3-none-any.whl (11 kB)
#11 5.780 Building wheels for collected packages: lxml, PyYAML
#11 5.780   Building wheel for lxml (setup.py): started
#11 5.933   Building wheel for lxml (setup.py): finished with status 'error'
#11 5.937   error: subprocess-exited-with-error
#11 5.937
#11 5.937   × python setup.py bdist_wheel did not run successfully.
#11 5.937   │ exit code: 1
#11 5.937   ╰─> [84 lines of output]
#11 5.937       Building lxml version 4.9.1.
#11 5.937       Building without Cython.
#11 5.937       Building against libxml2 2.9.10 and libxslt 1.1.34
#11 5.937       running bdist_wheel
#11 5.937       running build
#11 5.937       running build_py
#11 5.937       creating build
#11 5.937       creating build/lib.linux-aarch64-cpython-311
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/_elementpath.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/sax.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/doctestcompare.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/pyclasslookup.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/cssselect.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/usedoctest.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/ElementInclude.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/builder.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/defs.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/diff.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/html5parser.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/_html5builder.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/_setmixin.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/soupparser.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/usedoctest.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/_diffcommand.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/clean.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/builder.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/formfill.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       copying src/lxml/html/ElementSoup.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron
#11 5.937       copying src/lxml/isoschematron/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron
#11 5.937       copying src/lxml/etree.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/etree_api.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/lxml.etree.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/lxml.etree_api.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 5.937       copying src/lxml/includes/c14n.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/relaxng.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/uri.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xmlparser.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/etreepublic.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/dtdvalid.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/tree.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xmlerror.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/__init__.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xinclude.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/config.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xmlschema.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/htmlparser.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/schematron.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xpath.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/xslt.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/lxml-version.h -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       copying src/lxml/includes/etree_defs.h -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/rng
#11 5.937       copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/rng
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 5.937       copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 5.937       copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 5.937       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 5.937       running build_ext
#11 5.937       building 'lxml.etree' extension
#11 5.937       creating build/temp.linux-aarch64-cpython-311
#11 5.937       creating build/temp.linux-aarch64-cpython-311/src
#11 5.937       creating build/temp.linux-aarch64-cpython-311/src/lxml
#11 5.937       gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -I/usr/include/libxml2 -Isrc -Isrc/lxml/includes -I/usr/local/include/python3.11 -c src/lxml/etree.c -o build/temp.linux-aarch64-cpython-311/src/lxml/etree.o -w
#11 5.937       Compile failed: command 'gcc' failed: No such file or directory
#11 5.937       creating tmp
#11 5.937       cc -I/usr/include/libxml2 -I/usr/include/libxml2 -c /tmp/xmlXPathInit873kb263.c -o tmp/xmlXPathInit873kb263.o
#11 5.937       *********************************************************************************
#11 5.937       Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?
#11 5.937       *********************************************************************************
#11 5.937       error: command 'gcc' failed: No such file or directory
#11 5.937       [end of output]
#11 5.937
#11 5.937   note: This error originates from a subprocess, and is likely not a problem with pip.
#11 5.937   ERROR: Failed building wheel for lxml
#11 5.937   Running setup.py clean for lxml
#11 6.061   Building wheel for PyYAML (pyproject.toml): started
#11 6.240   Building wheel for PyYAML (pyproject.toml): finished with status 'done'
#11 6.240   Created wheel for PyYAML: filename=PyYAML-5.4.1-cp311-cp311-linux_aarch64.whl size=45659 sha256=839fc1ab47285af45c199736799509630fa901c2bcd8e1b35dcde867e6860c81
#11 6.241   Stored in directory: /root/.cache/pip/wheels/13/d5/5b/082ef0599cd0dde3d1f273ceebebe6cda67201058a70ade961
#11 6.242 Successfully built PyYAML
#11 6.242 Failed to build lxml
#11 6.276 Installing collected packages: webencodings, soupsieve, six, PyYAML, Markdown, lxml, bracex, backrefs, wcmatch, pymdown-extensions, html5lib, beautifulsoup4, pyspelling
#11 6.352   Running setup.py install for lxml: started
#11 6.499   Running setup.py install for lxml: finished with status 'error'
#11 6.514   error: subprocess-exited-with-error
#11 6.514
#11 6.514   × Running setup.py install for lxml did not run successfully.
#11 6.514   │ exit code: 1
#11 6.514   ╰─> [85 lines of output]
#11 6.514       Building lxml version 4.9.1.
#11 6.514       Building without Cython.
#11 6.514       Building against libxml2 2.9.10 and libxslt 1.1.34
#11 6.514       running install
#11 6.514       /usr/local/lib/python3.11/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
#11 6.514         warnings.warn(
#11 6.514       running build
#11 6.514       running build_py
#11 6.514       creating build
#11 6.514       creating build/lib.linux-aarch64-cpython-311
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/_elementpath.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/sax.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/doctestcompare.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/pyclasslookup.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/cssselect.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/usedoctest.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/ElementInclude.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/builder.py -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/defs.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/diff.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/html5parser.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/_html5builder.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/_setmixin.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/soupparser.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/usedoctest.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/_diffcommand.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/clean.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/builder.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/formfill.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       copying src/lxml/html/ElementSoup.py -> build/lib.linux-aarch64-cpython-311/lxml/html
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron
#11 6.514       copying src/lxml/isoschematron/__init__.py -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron
#11 6.514       copying src/lxml/etree.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/etree_api.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/lxml.etree.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/lxml.etree_api.h -> build/lib.linux-aarch64-cpython-311/lxml
#11 6.514       copying src/lxml/includes/c14n.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/relaxng.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/uri.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xmlparser.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/etreepublic.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/dtdvalid.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/tree.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xmlerror.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/__init__.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xinclude.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/config.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xmlschema.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/htmlparser.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/schematron.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xpath.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/xslt.pxd -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/lxml-version.h -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       copying src/lxml/includes/etree_defs.h -> build/lib.linux-aarch64-cpython-311/lxml/includes
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/rng
#11 6.514       copying src/lxml/isoschematron/resources/rng/iso-schematron.rng -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/rng
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 6.514       copying src/lxml/isoschematron/resources/xsl/XSD2Schtrn.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 6.514       copying src/lxml/isoschematron/resources/xsl/RNG2Schtrn.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl
#11 6.514       creating build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_dsdl_include.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_message.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_schematron_skeleton_for_xslt1.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_abstract_expand.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/iso_svrl_for_xslt1.xsl -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       copying src/lxml/isoschematron/resources/xsl/iso-schematron-xslt1/readme.txt -> build/lib.linux-aarch64-cpython-311/lxml/isoschematron/resources/xsl/iso-schematron-xslt1
#11 6.514       running build_ext
#11 6.514       building 'lxml.etree' extension
#11 6.514       creating build/temp.linux-aarch64-cpython-311
#11 6.514       creating build/temp.linux-aarch64-cpython-311/src
#11 6.514       creating build/temp.linux-aarch64-cpython-311/src/lxml
#11 6.514       gcc -pthread -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -I/usr/include/libxml2 -Isrc -Isrc/lxml/includes -I/usr/local/include/python3.11 -c src/lxml/etree.c -o build/temp.linux-aarch64-cpython-311/src/lxml/etree.o -w
#11 6.514       Compile failed: command 'gcc' failed: No such file or directory
#11 6.514       cc -I/usr/include/libxml2 -I/usr/include/libxml2 -c /tmp/xmlXPathInitndtq4698.c -o tmp/xmlXPathInitndtq4698.o
#11 6.514       *********************************************************************************
#11 6.514       Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?
#11 6.514       *********************************************************************************
#11 6.514       error: command 'gcc' failed: No such file or directory
#11 6.514       [end of output]
#11 6.514
#11 6.514   note: This error originates from a subprocess, and is likely not a problem with pip.
#11 6.516 error: legacy-install-failure
#11 6.516
#11 6.516 × Encountered error while trying to install package.
#11 6.516 ╰─> lxml
#11 6.516
#11 6.516 note: This is an issue with the package mentioned above, not pip.
#11 6.516 hint: See above for output from the failure.
------
executor failed running [/bin/sh -c pip3 install -r /requirements.txt]: exit code: 1
jonasbn@silverrocket:~/develop/github-collabs/spellcheck-github-actions
```

**Comment by jonasbn** [comment]
REF:

- https://docs.python.org/release/3.11.0/whatsnew/changelog.html

**Comment by jonasbn** [comment]
This PR (#132) demonstrating this issue, was surpassed by PR (#139).

It built perfectly, so this issue was solved via PR #139
